docs/alternate_domains.rst

```diff
@@ -1,12 +1,18 @@
 Alternate Domains
 =================
 
+.. note:: This documentation belongs to our community site.
```

Comment:
The note here should probably be something more clear. "These setup directions are for our community site" or something similar